lib/compiler.js

```diff
@@ -3,6 +3,7 @@
 const fs = require('fs')
 const solc = require('solc')
 const path = require('path')
+var findNodeModules = require('find-node-modules');
```

Comment:
I would be reluctant to include utility dependencies, is there any way to replace this with a shim?

Reply:
Probably with some recursive searching upwards. I will try, I needed a quick fix for my current project so this was fastest.